src/runtime/propensity-server.js

```diff
@@ -125,6 +159,28 @@ export class PropensityServer {
     return this.xhr_.fetch(url, init);
   }
 
+  /**
+   *
+   * @param {!../api/client-event-manager-api.ClientEvent} event
+   */
+  sendClientEvent_(event) {
```

Comment:
I would call this handleClientEvents_() since we are not always sending events.

Reply:
Ok